buildscripts/CMakeLists.txt

```diff
@@ -44,6 +44,10 @@ set(GL4ES_HASH SHA256=b565e717c7d192e936bda25f3cb90ad8db398af56414ec08294b671657
 set(OSG_VERSION 3.6.5)
 set(OSG_HASH SHA256=aea196550f02974d6d09291c5d83b51ca6a03b3767e234a8c0e21322927d1e12)
 
+# https://github.com/lz4/lz4
```

Comment:
The url should be pointing to releases, i.e. https://github.com/lz4/lz4/releases